### PR TITLE
Use latest dotnet sdk for builds.

### PR DESCRIPTION
**Pull request by codito**

### CODE REVIEW

File: scripts/build.ps1 [hunk 1]
```diff
@@ -61,8 +61,8 @@ $env:DOTNET_SKIP_FIRST_TIME_EXPERIENCE = 1
 # Dotnet build doesn't support --packages yet. See https://github.com/dotnet/cli/issues/2712
 $env:NUGET_PACKAGES = $env:TP_PACKAGES_DIR
 $env:NUGET_EXE_Version = "3.4.3"
-$env:DOTNET_CLI_VERSION = "2.1.0-preview1-006329"
-$env:DOTNET_RUNTIME_VERSION = "2.0.0-preview2-25331-01"
+$env:DOTNET_CLI_VERSION = "LATEST"
```
Comment (smadala) [inline scripts/build.ps1]: Update build.sh too.

File: scripts/build.ps1 [hunk 2]
```diff
@@ -296,15 +296,15 @@ function Publish-Package
...
+    # Copy Event Log Datacollector to Extensions folder.
     $eventLogDataCollector = Join-Path $env:TP_ROOT_DIR "src\DataCollectors\Microsoft.TestPlatform.Extensions.EventLogCollector\bin\$TPB_Configuration"
     $eventLogDataCollectorNetFull = Join-Path $eventLogDataCollector $TPB_TargetFramework
     Copy-Item $eventLogDataCollectorNetFull\Microsoft.TestPlatform.Extensions.EventLogCollector.dll $fullCLRExtensionsDir -Force
     Copy-Item $eventLogDataCollectorNetFull\Microsoft.TestPlatform.Extensions.EventLogCollector.pdb $fullCLRExtensionsDir -Force
     Copy-Item $eventLogDataCollectorNetFull\Microsoft.TestPlatform.Extensions.EventLogCollector.dll $coreCLRExtensionsDir -Force
     Copy-Item $eventLogDataCollectorNetFull\Microsoft.TestPlatform.Extensions.EventLogCollector.pdb $coreCLRExtensionsDir -Force
     
-	# Note Note: If there are some dependencies for the TestHostRuntimeProvider assemblies, those need to be moved too.
+    # Note Note: If there are some dependencies for the TestHostRuntimeProvider assemblies, those need to be moved too.
```
Comment (singhsarab) [inline scripts/build.ps1]: Nit: Please remove this extra Note.

File: NuGet.config [hunk 3]
```diff
@@ -1,15 +1,15 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
+  <packageRestore>
```
Comment (smadala) [inline NuGet.config]: If this makes restore implicit, make respective changes in `build.ps1.`

**Review by smadala — APPROVED** [review]

**Review by singhsarab — APPROVED** [review]

File: NuGet.config (hunk 3 above)
Comment (codito) [reply NuGet.config]: Fixed this

File: scripts/build.ps1 (hunk 1 above)
Comment (codito) [reply scripts/build.ps1]: Done

**Pull request merged**